java/libphonenumber/src/com/google/i18n/phonenumbers/Phonemetadata.java

```diff
@@ -0,0 +1,1119 @@
+/*
+ * Copyright (C) 2010 The Libphonenumber Authors
```

Comment:
No strong opinions from me, do you want us to :)? I would guess not, since I'm not the author and could not e.g. answer someone asking how this file was generated, e.g. at which version to run protoc to know what new methods should look like.